src/realtimecontroller/RealtimeController.ts

```diff
@@ -59,7 +59,35 @@ export default interface RealtimeController {
 
   /**
    * Subscribes to changes in attendee ids in order to discover attendee ids to
-   * subscribe and unsubscribe to for volume indicator updates.
+   * subscribe and unsubscribe to for volume indicator updates. The subscribed callback is called with below parameters:
+   *
+   * `attendeeId`:
+   * Unique `AttendeeId` received from `CreateAttendee` API representing an attendee.
+   *
+   * `present`:
+   * Indicates the attendee's presence in a meeting.
+   *
+   * `externalUserId`:
+   * Indicates the attendee's `externalUserId` provided while joining a meeting.
+   *
+   * `dropped`:
+   * Indicates whether the attendee dropped from the meeting. The Amazon Chime SDK for JavaScript reconnects a meeting session in below scenarios:
+   *
+   * - No audio packets (WebRTC)
+   * - Bad audio delay (WebRTC)
+   * - No pong reply (WebSocket)
+   *
+   * This value is provided by the Amazon Chime backend when an attendee is dropped and could not join the same meeting again due to re-connection issues.
+   * It is also provided to differentiate the scenarios between normal attendee leave and the attendee dropping due to re-connection issues.
+   *
+   * In re-connection scenarios, if an attendee drops and could never join back successfully, the JS SDK will call this callback setting
+   * the `dropped` value to a boolean value received from Amazon Chime backend, and it will set the `present` parameter to `false`.
+   *
+   * `posInFrame`: {@link RealtimeAttendeePositionInFrame}
+   *
+   * This object indicates which attendee out of how many total attendees the update is for.
+   * For example, if you were to join on a call with 3 total attendees, you would get presence callbacks for attendeeIndex 0, attendeeIndex 1, attendeeIndex 2 out of the total attendeesInFrame of 3.
+   * The first presence update you receive is for all of the present attendees and later presence updates are for individual attendees as they join and leave.
```

Comment:
>  The first presence update you receive is for all of the present attendees

Should we make it clear that the callback is trigger for each attendee once you join a meeting? This sounds like all of them at once.